toolkit/tools/pkg/imagecustomizerlib/cleanUpImage.go

```diff
@@ -0,0 +1,37 @@
+package imagecustomizerlib
+
+import (
+	"github.com/microsoft/azurelinux/toolkit/tools/internal/logger"
+	"github.com/microsoft/azurelinux/toolkit/tools/internal/safechroot"
+	"github.com/microsoft/azurelinux/toolkit/tools/internal/shell"
+	"github.com/sirupsen/logrus"
+)
+
+func cleanUpImage(imageChroot *safechroot.Chroot) error {
```

Comment:
nit - should this be `cleanTdnfCache` then now?

Reply:
updated name to `cleanTdnfCache`